x-pack/plugins/ml/common/openapi/ml_apis_v3.yaml

```diff
@@ -0,0 +1,169 @@
+openapi: 3.0.1
+info:
+  title: Machine learning APIs
+  description: Kibana APIs for the machine learning feature
+  version: "1.0.1"
+  license:
+    name: Elastic License 2.0
+    url: https://www.elastic.co/licensing/elastic-license
+tags:
+  - name: ml
+    description: Machine learning
+servers:
+  - url: https://localhost:5601/
+paths:
+  /s/{spaceId}/api/ml/saved_objects/sync:
+    get:
+      summary: Synchronizes Kibana saved objects for machine learning jobs and trained models.
+      description: >
+        You must have `all` privileges for the **Machine Learning** feature in the **Analytics** section of the Kibana feature privileges.
+        This API runs automatically when you start Kibana and periodically thereafter.
+      operationId: ml-sync
+      tags:
+        - ml
+      parameters:
+        - $ref: '#/components/parameters/spaceParam'
+        - $ref: '#/components/parameters/simulateParam'
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/mlSyncResponse'
+              examples:
+                syncExample:
+                  $ref: '#/components/examples/mlSyncExample'
+components:
+ parameters:
+  spaceParam:
+    in: path
+    name: spaceId
+    description: An identifier for the space. If `/s/` and the identifier are omitted from the path, the default space is used.
+    required: true
+    schema:
+      type: string
+  simulateParam:
+    in: query
+    name: simulate
+    description: When true, simulates the synchronization by returning only the list actions that would be performed.
```

Comment:
```suggestion
    description: When true, simulates the synchronization by returning only the list of actions that would be performed.
```

Reply:
Good catch!  Fixed in 533a87dfdf36958a6396803fca05f2997a736959